IPFS_WEEKLY_CALL_FORMAT.md

```diff
@@ -0,0 +1,57 @@
+# 🙌🏽 IPFS Weekly Call 📞 
+
+## Motivation
+
+For the past couple of months, we have been discussing revamping our Weekly Calls (formerly known as the IPFS All Hands ). If you want more context, please see this issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS Weekly Call to act as a space where we can showcase all of our hard work and game-changing projects. These meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the 🙌🏽 IPFS Weekly Call 📞  is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use these weekly calls to:
+
+- 1. Attract skilled contributors to your project
+- 2. Motivate the community about adding support for a new feature
+- 3. Find new partners, capabilities, or challenges for your project to work on/with
+- 4. Practice speaking in front of a captivated audience
+
+## The format
+
+The 30-minute IPFS Weekly Update takes place weekly on Mondays at:
+
+9:00 PST |  **17:00 UTC** | 18:00  CET
+
+Each week's call will feature one or more short talks from guest presenters, in one of the following formats:
+* One 15-minute talk (3x per month)
+* Three 5-minute lightning talks (1x per month)
+
+![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1541176158/ipfs_crew_xzbhxr.png)
+
+## List of upcoming talks  ⚡️📅⚡️
+
+[View the list of scheduled talks](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)
+
+## Steps to signing up and speaking
+
+- 1. Write a short blurb about your talk and propose a date, consulting the [calendar of upcoming talks](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) to see what dates are available. Email your proposal to portia @ protocol dot ai and you should receive an update about the status of your talk within 24-48 hours.
```

Comment:
Let's actually use a mlist for this proposals which Portia and others can subscribe to. Let me get that done

Reply:
created `ipfs-weekly-call-rfp@ipfs.io`